kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ClusterService.java

```diff
@@ -48,6 +48,13 @@ static String serviceName(Clusters cluster) {
         return cluster.getName();
     }
 
+    /**
+     * @return the fully qualified service hostname
+     */
+    static String absoluteServiceHost(KafkaProxy primary, Clusters cluster) {
+        return serviceName(cluster) + "." + primary.getMetadata().getNamespace() + ".svc.cluster.local";
```

Comment:
Doesn't this assume that we are only supporting in cluster access? Don't we need some way of determining the external name? 

or does that get layered on top somewhere else?

Reply:
In-cluster is our current objective. Off-cluster is for later, and you end up doing _a la_ Strimzi with things like `NodePort`, Load balancer and `Ingress`. Or I suppose to could also look at layering on top of Gateway API, decorator style.